low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -164,41 +161,37 @@ class SSPLDevDeploy:
         if rc != 0:
             raise DeployError(rc, "%s - %s CMD: %s",
                 out, "Unable to remove cortx-prereq RPMS. Cleanup failed.", cmd)
+        print('Cleaning up local repos...')
+        for p in Path('/etc/yum.repos.d').glob('var_artifacts_*'):
+            p.unlink()
+        for p in Path('/etc/yum.repos.d').glob('cortx-storage.colo.seagate.com_*'):
```

Comment:
cortx-storage.colo.seagate.com 
this was for internal repo right, why we need to use it now ? when we are removing references to these ? 
Again reference to internal domain name is not recommended

Reply:
We still allow specific cortx build deployment using sspl_dev_deploy. So cleanup required for that also.

Reply:
we allow means allowed outside seagate ? This guide is for community members, they would fail on this if access is denied ?

Reply:
No. for internal users. We have not mentioned in the guide, but we can still use internal builds to deploy.

Reply:
fine. @mukul-seagate11 do you see any concern?

Reply:
@thavanathan @cdeshmukh I feel we should avoid adding `cortx-storage`  anywhere in code. I guess we can clean up all local repositories other than OS repositories in the above case.

Reply:
thanks @shailesh-vaidya 

@thavanathan Spoke to Shailesh further and here is what we should address still
> we can clean all repos except default OS repositories
use `yum repolist` command to fetch currently configured repo's and remove newly added i.e. other than OS repo's
loop over repo names from `yum repolist` and remove them, except OS repos. that way we wont have to give the names 
can use name CentOS-7-OS to identify OS repos


>\# yum repolist
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-manager
Loading mirror speeds from cached hostfile
repo id                                                                                     repo name                                                                              status
!EOS_CentOS-7_CentOS-7-Extras                                                               CentOS-7-Extras                                                                           451
!EOS_CentOS-7_CentOS-7-OS                                                                   CentOS-7-OS                                                                            14,721
!EOS_CentOS-7_CentOS-7-Updates                                                              CentOS-7-Updates                                                                        3,329
!EOS_EPEL-7_EPEL-7                                                                          EPEL-7                                                                                 16,809
!EOS_Puppet6_puppet6-el7                                                                    puppet6-el7                                                                               184
docker-ce-stable/7/x86_64                                                                   Docker CE Stable - x86_64                                                                 122
repolist: 35,616

Reply:
@cdeshmukh @shailesh-vaidya as discussed I have changed the delete pattern to `cortx-`.